tests/run/type_inference.pyx

```diff
@@ -187,6 +187,8 @@ def arithmetic():
     assert typeof(f) == "long", typeof(f)
     g = 4 // <int>2
     assert typeof(g) == "long", typeof(g)
+    h = int(2) / 3.0
```

Comment:
PR changes also other operators (e.g. `*`). They are not tested here. Should not be added here?